packages/shared/components/popups/GovernanceCastVote.svelte

```diff
@@ -129,6 +130,8 @@
             void castVote()
         }
     }
+
+    const getSpinnerMessage = (): string => ($isSyncing ? 'general.syncing' : 'general.broadcasting')
```

Comment:
this is just a note, but I think `getSpinnerMessage` will not act in a reactive way, meaning, if while `disabled=true` then `$isSyncing` changes from false to true or the other way around, the spinner message will not reflect the change because the function does not depend on it, so the funcion will only be called on `disabled` changes. It is not important, we have the same issue on staking. Just wanted to point it out.
I hope I explained myself 😅

Reply:
Makes sense, I fixed it. I have a reactive dependency on `disabled`, which always sets the spinnerText anew.

Reply:
I think the changes are still not a perfect solution for the reactivity, because `disabled` can be updated to true > true > true based on different conditions and that will not trigger `setSpinnerMessage` because the boolean value didnt really change, svelte reactivity only triggers when oldvalue != new value.